tests/halt.s

```diff
@@ -1,5 +1,9 @@
 add $40 $2 #1
 add #1 $8 #2
 print #1
+mul $3 $2 #3
```

Comment:
You should update `coverage.s`, which stands for ISA coverage test item where `mul` is landed.